plugins/scanners/scanoss/src/main/kotlin/ScanOss.kt

```diff
@@ -48,6 +48,7 @@ import org.ossreviewtoolkit.utils.common.VCS_DIRECTORIES
 
 // An arbitrary name to use for the multipart body being sent.
 private const val FAKE_WFP_FILE_NAME = "fake.wfp"
+private const val ARG_FIELD_NAME     = "file"
```

Comment:
Style nit: Please don't align the `=`, but just use one single space before (and after) `=`.